### ISSUE TITLE
Date - Time - Place Component UI

### ISSUE THREAD

**Issue by MartinMelo97** [ActoSoft/MexicoTrasciende-Frontend#26]
Based on #12 mockup, create the component where is specified the date, time and the place for the event. 

**Aceptance criteria**
- Same icons as mockup
- Same shadow
- Same shape of the component

**Comment by MartinMelo97** [comment]
@MartinMelo97 Did you review the task?

**Comment by Cova14** [comment]
Do i have permissions to the mockups with this account? or with the Actosoft one?

**Comment by MartinMelo97** [comment]
In Drive with @actosoft.com.mx account

**Comment by Cova14** [comment]
Is it mobile mockup? I don't see any #12

**Comment by MartinMelo97** [comment]
Yes, its the white background row white the location, date and duration of the event, Did you see that?

**Comment by MartinMelo97** [comment]
@Cova14 don't worry. @UlisesGomezDW Is now working on this task.
